pkg/network/filter/packet_source_linux.go

```diff
@@ -151,10 +146,10 @@ func (p *AFPacketSource) pollStats() {
 				continue
 			}
 
-			p.polls.Add(sourceStats.Polls - prevPolls)
-			p.processed.Add(sourceStats.Packets - prevProcessed)
-			p.captured.Add(int64(socketStats.Packets()) - prevCaptured)
-			p.dropped.Add(int64(socketStats.Drops()) - prevDropped)
+			packetSourceTelemetry.polls.Add(sourceStats.Polls - prevPolls)
+			packetSourceTelemetry.processed.Add(sourceStats.Packets - prevProcessed)
+			packetSourceTelemetry.captured.Add(int64(socketStats.Packets()) - prevCaptured)
+			packetSourceTelemetry.dropped.Add(int64(socketStats.Drops()) - prevDropped)
```

Comment:
These stats seem like a great candidate for the prometheus-synced collection logic once we figure that out in a future PR.